server/sonar-web/src/main/js/apps/permission-templates/users-view.js

```diff
@@ -27,8 +27,8 @@ export default Modal.extend({
 
   onRender () {
     Modal.prototype.onRender.apply(this, arguments);
-    const searchUrl = '/api/permissions/template_users?ps=100&permission=' + this.options.permission.key +
-        '&templateId=' + this.options.permissionTemplate.id;
+    const searchUrl = window.baseUrl + '/api/permissions/template_users?ps=100&permission=' + 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)